test/externalTests/gnosis.sh

```diff
@@ -87,6 +87,12 @@ function gnosis_safe_test
     sed -i "s|it\(('can only be called from Safe itself'\)|it.skip\1|g" test/libraries/Migration.spec.ts
     sed -i "s|it\(('should enforce delegatecall to MultiSend'\)|it.skip\1|g" test/libraries/MultiSend.spec.ts
 
+    sed -i "s|it\(('is not called when setting initially'\)|it.skip\1|g" test/core/GnosisSafe.GuardManager.spec.ts
```

Comment:
This needs a comment explaining why we're disabling these tests and when the hack can be removed.